ansible/roles/products/common/docker-container/tasks/reinstall.yml

```diff
@@ -199,7 +199,7 @@
   ignore_errors: yes
   
 - name: Pulling latest hand_configurations
-  import_tasks: ../roles/products/common/docker-container/tasks/modify-container.yml
+  import_tasks: modify-container.yml
   vars:
     modify_script_name: "pull_sr_hand_config.sh"
-    modify_script_commands: "roscd sr_hand_config && git remote set-url origin https://github.com/shadow-robot/sr_hand_config && git pull"
+    modify_script_commands: "if roscd sr_hand_config; then roscd sr_hand_config git remote set-url origin https://github.com/shadow-robot/sr_hand_config && git pull; else roscd sr_config && git remote set-url origin https://github.com/shadow-robot/sr-config && git pull; fi"
```

Comment:
don't you miss a && between roscd sr_hand_config and git remote .. ?

Reply:
Thank you! Just added